manager/ui/apiman-plugin/plugins/apiman/html/user-services.html

```diff
@@ -7,46 +7,17 @@
   </head>
 
   <body>
-    
+  <body>
+    <div ng-controller="Apiman.UserServicesController">
```

Comment:
I think the controller should instead be set on the "page controller" div (see below).  The navbar ng-include has its own controller.